collector/container/konflux.Dockerfile

```
@@ -40,7 +40,9 @@ FROM scratch as builder
 
 COPY --from=rpm-implanter-builder /mnt /
 
-COPY . .
+ARG SOURCES_DIR=/staging
```

Optional, cosmetics, and up to you.

```suggestion
ARG SOURCES_DIR=/src
```

---

I think the chances of a directory in ubi existing called `/src` are slightly higher than `/staging`, so let's keep it like that.